src/components/home/Section.tsx

```diff
@@ -1,9 +1,15 @@
 import React from "react";
+import ListSection from './sections/ListSection';
+import { MORE_MOVIES, MOVIES } from '../../mocks/mocks';
+import PopularListSection from './sections/PopularListSection';
 
 const Section: React.FC = () => {
-    return (
-        <div>SectionComponent</div>
-    );
-}
+	return (
+		<div>
+			<ListSection movies={MOVIES} />

```

Comment:
Co w przypadku, gdy API zawiedzie, rzuci 500 i przykładowo `MOVIES`, `MORE_SERIES` nie będą mieć przypisanej wartości.
Czy w takiej konfiguracji jest to w ogóle obsłużone, czy zacznie wyrzucać błędy w konsoli i component'y przestaną działać.

Reply:
> Co w przypadku, gdy API zawiedzie, rzuci 500 i przykładowo `MOVIES`, `MORE_SERIES` nie będą mieć przypisanej wartości. Czy w takiej konfiguracji jest to w ogóle obsłużone, czy zacznie wyrzucać błędy w konsoli i component'y przestaną działać.

no teraz to jest mock po prostu, jak będzie api stało to się zmieni trochę kod i będzie pusta tablica domyślnie po prostu

Reply:
Czasami warto od razu takie rzeczy ustawić, by później nie zapomnieć. A znając każdego programistę napewno zapomnimy. Ja bym od razu wstawił dla świętego spokoju.